public/app/features/alerting/unified/api/notificationPoliciesApi.ts

```diff
@@ -1,11 +0,0 @@
-import { generatedRoutesApi } from 'app/features/alerting/unified/openapi/routesApi.gen';
-
-export const routingTreeApi = generatedRoutesApi.enhanceEndpoints({
-  endpoints: {
-    replaceNamespacedRoutingTree: {
-      // Stop a failed mutation from invalidating the cache, as otherwise the notification policies
```

Comment:
@tomratcliffe I removed this since I assumed we no longer need it now that we have stable identifiers for the routes – I tested this for deleting routes and it does seem to work without this workaround.
Is there another scenario other than the one described here where we want to _not_ invalidate the cache?

Reply:
I had added this because when the mutation fails, we still refetch the policy tree, which results in the user losing the changes they may have made in the edit modal. I know that regardless you have to reload the page, but I wanted to preserve what the user had put into the form so they don't "lose" it. I thought this was the case on `main`, but I checked and apparently not... So it seems that this is still functioning as it already was :+1: